ic-kit/Cargo.toml

```diff
@@ -1,25 +1,26 @@
 [package]
 name = "ic-kit"
-version = "0.5.0-alpha.0"
+version = "0.5.0-alpha.1"
```

Comment:
There's some export issue on the release, and I tried to push a new version but was still broke, so I yanked it. So should use .2